code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -71,7 +71,11 @@
 
 /obj/item/clothing/suit/armor/reactive/teleport/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
 	if(!active)
-		return 0
+		return FALSE
+	if(isprojectile(hitby))
+		var/obj/item/projectile/P = hitby
+		if(P.martial_arts_no_deflect)
+			return FALSE
```

Comment:
Can you make this into a proc on the reactive armour like can_trigger() to reduce code duplication?

Reply:
yeah, I'm not sure why I didn't think to do that in the first place. I know better than to copy/paste identical code like I did lol